R/gh.R

```diff
@@ -208,6 +211,24 @@ gh <- function(endpoint, ..., per_page = NULL, .token = NULL, .destfile = NULL,
   res
 }
 
+gh_response_length <- function(res) {
+  if (!is.null(names(res)) && length(res) > 1 &&
+      names(res)[1] == "total_count") {
+    # Ignore total_count, incomplete_results, repository_selection
+    # and take the first list element to get the length
+    lst <- vapply(res, is.list, logical(1))
+    nm <- setdiff(
+      names(res),
+      c("total_count", "incomplete_results", "repository_selection")
+    )
+    tgt <- which(lst[nm])[1]
```

Comment:
Include some kind of runtime assert that checks that there is only one target? If not throw an error and ask the user to report the endpoint?

Reply:
Well, you PR merges all elements, and I looked over the spec, so this should be fine, I think. If there are multiple of them, we'd still want to do something, so I think taking the first is fine.

Reply:
Makes sense, particularly if we're just looking for the length